charts/stable/hastebin/chart/values.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/k8s-at-home/haste
+  pullPolicy: IfNotPresent
+  tag: vce75235
```

Comment:
Does this tag exist?

Reply:
Use to, may not anymore,
removed duplicate